test/tools/ossfuzz/yulFuzzerCommon.cpp

```diff
@@ -23,7 +23,7 @@ using namespace yul::test::yul_fuzzer;
 void yulFuzzerUtil::interpret(ostream& _os, shared_ptr<yul::Block> _ast)
 {
 	InterpreterState state;
-	state.maxTraceSize = 10000;
+	state.maxTraceSize = 75;
```

Comment:
Isn't that a bit too short?

Reply:
That's what I felt in the beginning but looking at the data, almost all test inputs (including crash inducing ones) have a trace size under 50. The inputs that require larger traces are usually nested for loops with hundreds of inner loops.

Reply:
Ok!